tests/integration/test_hedged_requests/configs/logger.xml

```diff
@@ -0,0 +1,5 @@
+<clickhouse>
+    <logger>
+        <count>20</count>
```

Comment:
By default we have max 10 log files, but tests in test_hedged_requests restart server more than 10 times. Let's increase it to be able to see all logs when tests are failed